roofit/xroofit/src/xRooNode.cxx

```diff
@@ -2092,7 +2100,7 @@ xRooNode xRooNode::Add(const xRooNode &child, Option_t *opt)
       child.convertForAcquisition(*this);
       if (child.get()) {
          if (auto _d = child.get<RooAbsData>()) {
-            // don't use acquire method to import, because that adds datasets as Embedded
+            // don't use acquire method to import, because that adds datasets as Embeddded
```

Comment:
```suggestion
            // don't use acquire method to import, because that adds datasets as Embedded
```
I guess this typo can stay fixed, but it also doesn't matter. I would like to merge this PR quickly. But the the next update at least :slightly_smiling_face: